app/assets/stylesheets/player.scss

```diff
@@ -321,9 +318,9 @@ body {
     }
 
     .playlist-element {
-      margin: 0;
+      margin: 0em;
```

Comment:
Properties should be ordered color, font-size, margin, padding<br>`0em` should be written without units as `0`